src/features/cartographie/components/lieux-mediation-numerique-details/lieux-mediation-numerique-details.component.html

```diff
@@ -164,6 +162,23 @@ <h2 class="h5 text-muted text-uppercase mt-1 flex-grow-1">Une erreur sur les don
       <app-webinaire></app-webinaire>
     </div>
   </div>
+  <div class="flex-column bg-light pt-2 pb-4 d-sm-none d-flex">
+    <small class="mt-4 mb-2 mx-4 fw-bold">
+      Vous travaillez dans cette structure ou êtes habilités à
+      <br />
+      mettre à jour les données de cette fiche ? Si besoin, nous
+      <br />
+      organisons des webinaires mensuels pour vous
+      <br />
```

Comment:
```suggestion
      <br aria-hidden="true" />
      mettre à jour les données de cette fiche ? Si besoin, nous
      <br aria-hidden="true" />
      organisons des webinaires mensuels pour vous
      <br aria-hidden="true" />
```
D'après ce que j'ai lu, les <br/> sont pris en compte par les lecteurs d'écrans et traités comme des fins de phrases, donc dans ce cas ça risque de causer des ruptures dans la lecture à des endroits non souhaités.